docs-web/Makefile

```diff
@@ -0,0 +1,21 @@
+NETLIFY_ROOT = /opt/build/repo
+SITE_ROOT = ${NETLIFY_ROOT}/site
+BUILDDIR = ${NETLIFY_ROOT}/docs-web/build
+THEME_REPO = ${THEME_REPO_URL}
+
+.PHONY: all build clean setup
+
+all: clean setup build
+
+clean:
+	rm -rf ${SITE_ROOT}
+
+setup:
+	mkdir ${SITE_ROOT}
```

Comment:
```suggestion
	mkdir -p ${SITE_ROOT}
```

Reply:
@lucacome Thanks! The Netlify build environment always clones the repo into the path `/opt/build/repo`. If we're just adding another directory in that existing location, do we need to use the `-p`?

Reply:
It's just safer, if the directory already exists or if part of the path doesn't exist, it won't throw an error. But not strictly necessary, that's why I approved anyway 🙂

Reply:
I agree with @lucacome here, `-p` is considered good practice and certainly won't do any harm but it's not necessarily needed.

Reply:
Thanks guys!